index.html

```diff
@@ -1 +1 @@
-<h1> Hello Harman! <h1>
+<h1> harman is cute <h1>
```

Comment:
I think there was a mistake in the previous code.
Please close the h1 block
```suggestion
<h1> Harman is cute. </h1>
```